src/Controller/Admin/UserCrudController.php

```diff
@@ -15,7 +17,7 @@
 
 class UserCrudController extends AbstractCrudController
 {
-    public function __construct(private TranslatorInterface $translator)
+    public function __construct(private TranslatorInterface $translator, private MunicipalityRepository $municipalityRepository)
```

Comment:
Is `$this->municipalityRepository` used anywhere?

Reply:
It is not, removed.